code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/SmirnovAndrewTest.java

```diff
@@ -0,0 +1,49 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+import static org.junit.Assert.*;
```

Comment:
It is unnesessary to use * in import, choose exact functions instead

Reply:
fixed